Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,524 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt20">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    Masḥafa qəddāse
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 20</idno>
+                        <altIdentifier><idno>Löfgren 17</idno></altIdentifier>
+                        <altIdentifier><idno>Löfgren V</idno></altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="192vb"/>
+                            <title type="complete" ref="LIT1960Mashaf"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1ra" to="5va"/>
+                                <title type="complete" ref="LIT1960Mashaf#PrayerKidan"></title>
+                                
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="1ra" to="3ra"/>
+                                    <title type="complete" ref="LIT1960Mashaf#Kidan1"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.2">
+                                    <locus from="3ra" to="4va"/>
+                                    <title type="complete">Kidān za-qatr</title>
```

Comment:
If "Kidān za-qatr" is a subchapter of `LIT1960Mashaf#Kidan1` it is also possible to create a div (nested under the `div `with xml:id `Kidan1`) under `LIT1960Mashaf` and the label Kidān za-qatr, then one can refer from here and other similar mss to LIT1960Mashaf#Kidanzaqatr (pr whichever the new xml:id would be)

Reply:
Looking into this, I do not understand on the basis of what Löfgren analysed this as a separate part. It has no marked beginning in the manuscript (as he notes) and he only refer to Dillmann's Chrestomathia, where it is defined as part of Kidān za-nagh. I think I will encode it simply as Kidān za-nagh and add a note about Löfgren's analysis, if that sounds ok.